packages/devtools-launchpad/src/index.js

```diff
@@ -104,7 +105,19 @@ function bootstrap(React, ReactDOM, App, appActions, appStore) {
   }
 
   const { store, actions, LandingPage } = initApp();
-  renderRoot(React, ReactDOM, LandingPage, store);
+
+  let landingPage = connect(
```

Comment:
this should still be in the `LandingPage` component.

you can do something like 
`const { wrappedComponent: LandingPage } = require("components/LandingPage")`